src/components/pages/ProfileList/ProfileListContainer.js

```diff
@@ -9,8 +8,8 @@ import RenderProfileListPage from './RenderProfileListPage';
 
 // Here is an example of using our reusable List component to display some list data to the UI.
 const ProfileList = () => {
-  const { authState } = useOktaAuth();
-
+  //Variable is created as place-holder. Function destroyed while removing Okta
```

Comment:
I think this is a typo. you need a space after `//` in order for the compiler to see it as commented out.